tests/test_grid.py

```diff
@@ -26,3 +26,14 @@ def test_get_grid_scrip():
     ds_test = pop_tools.get_grid('POP_gx3v7', scrip=True)
     ds_ref = xr.open_dataset(DATASETS.fetch('POP_gx3v7.nc'))
     assert ds_compare(ds_test, ds_ref, assertion='allclose', rtol=1e-14, atol=1e-14)
+
+
+def test_get_grid_to_netcdf():
+    for grid in pop_tools.grid_defs.keys():
+        print('-' * 80)
+        print(grid)
+        ds = pop_tools.get_grid(grid)
+        for format in ['NETCDF4', 'NETCDF3_64BIT']:
+            gridfile = f'{grid}_{format}.nc'
+            ds.to_netcdf(gridfile)
```

Comment:
```suggestion
            ds.to_netcdf(gridfile, format=format)
```

Reply:
Oooh!  Good catch.  This will probably break the tests and force me to implement the `char`-type solution.